sentinel/sentinel.go

```diff
@@ -163,22 +163,22 @@ func (p Proxy) handleClaim(w http.ResponseWriter, r *http.Request) {
 	path := r.URL.Path
 
 	parts := strings.Split(path, "/")
-	if len(parts) < 1 {
+	if len(parts) < 3 {
```

Comment:
confused by this, what are the other parameters? It looks like we are just using contractId, so upstream callers should be fixed rather than this handler?

Reply:
https://github.com/arkeonetwork/arkeo/blob/00e0842b0e0059c0e63dc56f5ec31d82a29d17d1/tools/curleo/main.go#L140

Reply:
yep same as above it parses `/claim/123` as ["","claim","123"]